clients/js/lib/stencil.js

```diff
@@ -0,0 +1,88 @@
+const protobuf = require('protobufjs');
```

Comment:
Any reason import for this is smallase while In other caps. Let's try to keep it consistent.

Reply:
we're using `node-fetch` which is nodejs port for browser fetch API. Want to keep import for fetch as `fetch` instead of `Fetch` to keep the API similar. All external imports are in small cases. Our exports (Stencil, MultiURLStencil) are in capital cases. we can move these as well if we want.

Reply:
Not a big deal but let's keep consistent.